src/models/init.js

```diff
@@ -29,6 +29,7 @@ const knexConfig = {
       user: process.env.POSTGRES_USER,
       password: process.env.POSTGRES_PASSWORD,
       database: process.env.POSTGRES_DB,
+      ssl: { rejectUnauthorized: false }
```

Comment:
Hey, we can't merge this change since this disables strict SSL checks and bypasses security.

Reply:
I agree. What is missing in the current code base is the configuration support for using ssl enabled postgres database. Instead of hardcoding the value to false directly, we parameterized it so that it can be controlled during the deployment based on the requirement.

Reply:
Hey we've moved away from this old release and have deprecated this release. Please migrate to our newer codebase which exists on master. We've also stopped actively supporting this build and this would be our last code merge to this build.

Please let us know if we can connect to help your team migrate to the newer codebase.